ecllibrary/std/Uni.ecl

```diff
@@ -398,4 +398,17 @@ EXPORT unicode GetNthWord(unicode text, unsigned4 n, varstring localename = '')
 EXPORT ExcludeNthWord(unicode text, unsigned4 n, varstring localename = '') :=
     lib_unicodelib.UnicodeLib.UnicodeLocaleExcludeNthWord(text, n, localename);
 
+/**
+ * Returns the source string with the all characters that match characters in the search string replaced
+ * with the character at the corresponding position in the replacement string.
```

Comment:
If the string object has an invalid sequence, then isEmpty() will be true.  The explicit test is isBogus().  Since you did not know that and presumably a casual reader will not know that, it would be better to add a comment that your isEmpty() test also tests for invalid sequences.